src/components/queryBuilder/utils.ts

```diff
@@ -255,6 +255,8 @@ export const getSQLFromQueryOptions = (options: SqlBuilderOptions): string => {
       query += getGroupBy(options.groupBy, options.timeField);
       break;
     case BuilderMode.List:
+    case BuilderMode.Log:
+      options.fields = ['*'];
```

Comment:
I fixed the linting errors. The tests are now failing because when `options.mode` is `BuilderMode.List` it will set `options.fields` to `*`.
I think it would be good to have a few tests in Log mode. 😃

Reply:
i know this. but it don't be set in master. so I don't set it.
In fact. if you select show as list, the sql will be error.
If necessary. I can add it.